R/graph_theme.R

```diff
@@ -0,0 +1,11 @@
+jrc_theme <- function() {
+    theme(
+        plot.title = element_text(hjust = 0.5),
+        panel.border = element_rect(fill = NA),
+        axis.line = element_line(),
+        text = element_text(size = 11, family = "Georgia"),
```

Comment:
Did you try this on all your computers? I'm not sure this font exists everywhere.

Reply:
We did it on Farhan's Mac and it worked. We did not try any other computer.

Reply:
That should be fine for now. If there are problems later on, we can help figure it out. Enjoy your fall break!